src/common/modules/UnicodeFontHandler.js

```diff
@@ -0,0 +1,245 @@
+"use strict";
+
+import * as AddonSettings from "/common/modules/AddonSettings/AddonSettings.js";
+// import * as BrowserCommunication from "/common/modules/BrowserCommunication/BrowserCommunication.js";
+import { isMobile } from "./MobileHelper.js";
+
+import { COMMUNICATION_MESSAGE_TYPE } from "/common/modules/data/BrowserCommunicationTypes.js";
+import { fonts } from "/common/modules/data/Fonts.js";
+
+const caseIds = Object.freeze(["Lowercase", "Uppercase", "Capitalize Each Word", "Toggle Case"]);
+const fontIds = Object.freeze(["Superscript", "Small Caps", "All Small Caps", "Unicase", "separator", "Serif bold", "Serif italic", "Serif bold italic", "Sans-serif", "Sans-serif bold", "Sans-serif italic", "Sans-serif bold italic", "Script", "Script bold", "Fraktur", "Fraktur bold", "Monospace", "Double-struck", "separator", "Circled", "Circled (black)", "Squared", "Squared (black)", "Fullwidth"]);
+
+// Thunderbird
+const THUNDERBIRD = typeof messenger !== "undefined";
+
+// Chrome
+const CHROME = Object.getPrototypeOf(browser) !== Object.prototype;
+
+/**
+ * Change Unicode font.
+ *
+ * @param {string} atext
```

Comment:
I usually don't see sense in prepending variables with `a` . If it is a reserved keyword, you'll notice this.
If not, we have proper scoping in JS modules, so just use ` text` here.